README.md

```diff
@@ -195,7 +193,23 @@ docker run \
     apache/age
 ```
 
+<h5> Or Run docker container if you previously worked with age container </h5>
```

Comment:
Also, capitalize the `age` keyword here

Reply:
the container name is case sensitive... Container won't run otherwise

Reply:
This is only a task description so you can capitalize the alphabet here. The main script is line 199 which should be case sensitive.

Reply:
updated